doc/sphinx/source/tutorials/thcov_tutorial.rst

```diff
@@ -1,10 +1,11 @@
 .. _thcov_tutorial:
+
 How to include a theory covariance matrix in a fit
 ==================================================
 
 This section details how to include :ref:`scale variation covariance matrices (covmats) <vptheorycov-index>`
-in a PDF fit. At the present time this can only be done at next-to-leading order (NLO), for which the
-central theory is :ref:`theory 163 <theory-indexes>`.
+in a PDF fit.
+This can only be done with specific theories for which the theory uncertainties are defined.
```

Comment:
```suggestion
```

I suppose this is explained in the vptheorycov-index and anyway is not a very clear statement on its own

Reply:
The docs for the theory covmat stuff is quite behind I'd say.